### ISSUE TITLE
Can not build with clang-cl due to /MP

### ISSUE THREAD

**Issue by emmenlau** [vxl/vxl#863]
I'm trying to build ITK with clang-cl, the LLVM compiler frontend for the MSVC compiler. The build fails due to the use of `/MP` on the command line. I could trace this down to:
https://github.com/vxl/vxl/blob/c6c899aaf9cbf7ccabfca1ba35b2248fb611ffbc/config/cmake/config/vxl_utils.cmake#L89

Since this flag is not supported (at least not in its current form) by clang-cl, it should not be added on this compiler combination. I tested that the following check works (the current cmake-recommended way to detected clang-cl):
```cmake
if(MSVC AND NOT vxl_add_DISABLE_MSVC_MP AND NOT "${CMAKE_CXX_SIMULATE_ID}" STREQUAL "MSVC")
```

Would that be possible to add?

**Comment by emmenlau** [comment]
Anyone?

**Comment by emmenlau** [comment]
Is this fixed by https://github.com/vxl/vxl/commit/47490550d63a4ca570a20a9d70244e233f06c1bd?

**Comment by emmenlau** [comment]
Closing as this is presumably fixed by 4749055
